charts/promitor-scraper/templates/_helpers.tpl

```diff
@@ -0,0 +1,43 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
What is this vim setting for?

Reply:
Comes default when you create a helm chart